client/app/components/saveable-form.hbs

```diff
@@ -6,12 +6,12 @@
   )
 
   saveButton=(component 'packages/save-button'
-    onClick=this.save
+    _saveChangeset=(fn this.saveChangeset this.saveableChanges)
```

Comment:
That's odd that the underscore indicate `_saveChangeset` is an internal property, but it's being referenced as a public parameter.

Reply:
It's private to saveable-form